themes/default/layouts/page/aws-summit-2022.html

```diff
@@ -0,0 +1,125 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section id="ctas" class="container mx-auto my-16 text-center">
+        <h3>July 19 - NYC and August 24 - Chicago</h3>
+        <h2>Interact with us at AWS Summit 2022!</h2>
+        <div class="my-16">
+            <div class="lg:flex lg:flex-wrap justify-center items-stretch">
+                <div class="lg:w-1/2 px-3 py-6">
```

Comment:
I think if add `h-full` to this parent div the cards will all stretch to the same height.

Also this page is likely to be consumed on mobile mostly so not the biggest deal.